airbyte-cdk/python/README.md

```diff
@@ -65,7 +65,7 @@ pip install -e ".[dev]" # [dev] installs development-only dependencies
 If the iteration you are working on includes changes to the models, you might want to regenerate them. In order to do that, you can run:
 
 ```bash
-./gradlew :airbyte-cdk:python:format
+./gradlew :airbyte-cdk:python:build
```

Comment:
That seems like a docs mistake from previous things, right?

Reply:
Yes, gradle `format` tast does not exist anymore. `airbyte-ci format fix` should be used instead